src/web_app/html/index_template.html

```diff
@@ -110,6 +110,10 @@
     </svg>
 
   </div>
+  <div id="privacy">
+    <span><a href="//www.google.com/accounts/TOS">Terms</a></span>
```

Comment:
The problem is that footer element is only shown when joining the room, we only want to display the links on the room joining page.